app/src/main/java/app/grapheneos/camera/CamConfig.kt

```diff
@@ -894,6 +898,14 @@ class CamConfig(private val mActivity: MainActivity) {
         startCamera(true)
     }
 
+    // Will always return a valid CameraInfo if called within startCamera
+    private fun getCurrentConfigCameraInfo() : CameraInfo? {
+        for (cameraInfo in cameraProvider!!.availableCameraInfos) {
+            if (cameraInfo.lensFacing == lensFacing) return cameraInfo
```

Comment:
There can be more than one camera with same lensFacing value, correct? This code doesn't account for that

Reply:
Our app only supports handling a single camera for each `lensFacing` currently. We could stick to saving info of a single camera for each side (into variables `frontCameraInfo` and `rearCameraInfo`) and use the physical camera ID present in that info to select and display information about the right camera to avoid ambiguity just in case if the order in which availableCameras are returned is uncertain/could change in the future

Reply:
Our camera app currently only supports using only one camera for each side facing which is chosen by the CameraX library. However, considering the possibility that the way CameraX chooses the camera could differ from our chosen camera info (now/later in case of multiple cameras), we could explicitly choose a cameraInfo for both front and rear facing (if they are available) and ensure that the app only chooses that camera we're showcasing the information for. Support for selection of camera can be dealt with a separate PR as our app currently displays the camera chosen by CameraX by default.